lib/IAMClient.js

```diff
@@ -52,6 +53,19 @@ class IAMClient {
         }
         this.accessKey = accessKey;
         this.secretKeyValue = secretKeyValue;
+        this.log = new Logger('IAMClient', {
```

Comment:
We should call this VaultClient throughout.  I know the file is IAMClient.js, but we always refer to this module as vaultClient and it does have the authentication routes in addition to true "IAM" functions.